### PR TITLE
Remove travis jobs now running in GHA

### PR DESCRIPTION
**Pull request by paulb777**
#no-changelog

### CODE REVIEW

File: .github/workflows/core.yml [hunk 1]
```diff
@@ -19,6 +19,35 @@ jobs:
...
     - name: Build and test
-      run: ./scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }}
+      run: scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }}
+
+  core-use-libraries:
+    runs-on: macos-latest
+    if: github.event_name == 'schedule'
+    strategy:
+      matrix:
+        target: [ios, tvos, macos]
```
Comment (wilhuff) [inline .github/workflows/core.yml]: You can have just one definition if you use the matrix to multiply out the combinations:

```yaml
strategy:
  matrix:
    target: [ios, tvos, macos]
    flags: [
      '--skip-tests --use_modular_headers',
      '--skip-tests --use_libraries'
    ]
```
Then below
```yaml
run: scripts/third_party/travis/retry.sh ./scripts/pod_lib_lint.rb FirebaseCore.podspec --platforms=${{ matrix.target }} ${{ matrix.flags }}

**Comment by paulb777** [comment]
Thanks @wilhuff!

cc: @ryanwilson @maksymmalyhin @mikehaney24 @schmidt-sebastian @dmandar

File: .github/workflows/core.yml (hunk 1 above)
Comment (mikehaney24) [reply .github/workflows/core.yml]: @paulb777  --use-modular-headers and --use-libraries

Comment (paulb777) [reply .github/workflows/core.yml]: Yep - thanks! #4853

**Pull request merged**